src/schema/artwork/index.ts

```diff
@@ -206,7 +206,26 @@ export const artworkFields = () => {
     },
     edition_sets: {
       type: new GraphQLList(EditionSet.type),
-      resolve: ({ edition_sets }) => edition_sets,
+      args: {
+        sort: EditionSetSorts,
+      },
+      resolve: ({ edition_sets }, { sort }) => {
+        if (sort) {
+          // Only ascending price sort supported currently.
```

Comment:
Nice!

Is it possible to specify an unknown sort option from the client as this is currently structured? If so, would this return edition sets sorted by ascending price or unsorted?